lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,7 +35,9 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
+          if cached_template = find_cached_template
```

Comment:
:+1: 

I've removed the old implementation. To get the tests passing I needed to explicitly reload the configuration.